src/foam/nanos/ruler/RulerDAO.js

```diff
@@ -49,6 +45,7 @@ foam.CLASS({
       of: 'foam.mlang.predicate.Predicate',
       name: 'createBefore',
       javaFactory: `return AND(
+  EQ(Rule.ASYNC, false),
```

Comment:
More of a pet peeve but the indenting in the javaFactories for this model look off

Reply:
The indentation is for the generated java code.